src/flamegpu/sim/SimRunner.cu

```diff
@@ -125,11 +127,13 @@ void SimRunner::start() {
                 }
                 return;
             } else {
-                if (verbose) {
+                // Progress flush
+                if (verbosity == VERBOSE) {
```

Comment:
If above is changed, this will need updating too.

Reply:
Agreed